docs/changelog.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../CHANGELOG.md
```

Comment:
I think this will fail because it is a literal include of markdown syntax in an RST document. You may need to rename it to `changelog.md` and use the markdown version of the syntax:

    ```{include} ../CHANGELOG.md
    ```